app/assess/models/flag_teams.py

```diff
@@ -61,6 +62,13 @@ def from_flags(cls, flags_list: List[FlagV2], teams_list: list = None):
                     elif flag.latest_status == FlagTypeV2.STOPPED:
                         num_of_stopped = num_of_stopped + 1
 
+            if num_of_flags > 0:
+                for flag_count in range(num_of_flags):
+                    ordinal = num2words(
+                        flag_count + 1, to="ordinal"
+                    ).capitalize()
+                    ordinal_list.append(ordinal)
+
```

Comment:
similar implementation is already commented on line 57(https://github.com/communitiesuk/funding-service-design-assessment/pull/359/files#diff-1def9e84c27fe41df2e014cd2642e3419d196fad97e3325020bb6caa5969fbafR57) . Can you just add your code there, it reduces the num of for loops & checks